src/SEOManager.php

```diff
@@ -176,6 +178,36 @@ public function flipp(string $alias, string|array $data = null): string|static
         return $this->set('image', "https://s.useflipp.com/{$template}.png?s={$signature}&v={$query}");
     }
 
+    /** Enable favicon extension. */
+    public function favicon(string $path): static
+    {
+        $this->extensions['favicon'] = true;
```

Comment:
Here we could do something like:

```suggestion
        if (! class_exists(ImageManager::class)) {
            throw new Exception('Intervention not available, please run composer require intervention/image');
        }

        $this->extensions['favicon'] = true;
```

Reply:
How are we gonna test this?

Reply:
Add the dependency to composer-dev, and the case when it's not defined will just be untested now I guess. There might be some way to mock that or undefine the class during runtime, but it's fine to not test that single `if ()` statement.